terraform/modules/app-container-definition/main.tf

```diff
@@ -0,0 +1,49 @@
+variable "name" {
+  type = string
+}
+variable "image" {
+  type = string
+}
+variable "environment_variables" {
+  type = map
+}
+variable "log_group" {
+  type = string
+}
+variable "secrets_from_arns" {
+  type = map
+}
+variable "aws_region" {
+  type = string
+}
+output "value" {
+  value = {
+    "name" : var.name,
+    "image" : var.image,
+    "essential" : true,
+    "environment" : [for key, value in var.environment_variables : { name : key, value : value }],
+    "dependsOn" : [{
+      "containerName" : "envoy",
+      "condition" : "START"
+    }],
+    "logConfiguration" : {
+      "logDriver" : "awslogs",
+      "options" : {
+        "awslogs-create-group" : "true",
+        "awslogs-group" : var.log_group,
+        "awslogs-region" : var.aws_region,
+        "awslogs-stream-prefix" : var.name,
+      }
+    },
+    "mountPoints" : [],
+    "portMappings" : [
+      {
+        "containerPort" : 80,
+        "hostPort" : 80,
+        "protocol" : "tcp"
+      }
+    ],
```

Comment:
Suspect we might end up needing the ability to pass these in a well, but I think hardcoding 80 for now is ok — let's see how it goes. Maybe it'll discourage people from doing whacky things with the port mappings this time 😛

Reply:
Yeah, I think these also have to match the envoy configuration:

https://github.com/alphagov/govuk-infrastructure/pull/102/files#diff-21c98290833886dad6304137790d6ef2775d8ed580f91b4fe77690c380d762f2R52

... so there's a bit more coupling there. Hopefully we won't have to make this configurable for the GOV.UK apps. 

Router is a possible exception, since that wants to listen on two ports at once at the moment - maybe we can fix that when we change how it loads its config. For now, we can always use `merge()` on the output of this module if we really want to customise something in a one-off cast.